dpe/src/commands/derive_context.rs

```diff
@@ -996,4 +1042,105 @@ mod tests {
         let digest = hasher_2.finish().unwrap();
         assert_eq!(digest.bytes(), dpe.contexts[child_idx].tci.tci_cumulative.0);
     }
+
+    #[test]
+    fn test_cdi_export_flags() {
```

Comment:
nit: It would be helpful to have a quick comment above each different edge case that's being tested.